src/plugins/imagePlugin.js

```diff
@@ -55,9 +55,13 @@ export default function createImagePlugin() {
     const licenseCopyString = `${license.toLowerCase().includes('by')
       ? 'CC '
       : ''}${license}`.toUpperCase();
-    const authorsCopyString = authors
-      .filter(author => author.type !== 'Leverandør')
-      .map(author => `${author.name}`)
+
+    const authorsWithoutSuppliers = authors.filter(
+      author => author.type !== 'Leverandør'
```

Comment:
Hmm, var det ikke slik at bare opphavsmann skulle vises på bilde? Om så er det kanskje lurere å filtrere på typer som telles som opphavsmann i stedet for å bare filtrere bort bare "Leverandør"? Mener å huske at @runesto hadde et dokumnet som viser hvilke mapping mellom gamle og nye typer

Reply:
Både fotograf og opphavsmann er brukt som type (går ut i fra at fotograf skal vises). Så trenger egentlig en liste med hvilke typer som skal vises isåfall.

Reply:
https://docs.google.com/document/d/1UpQxxWzA8GIL1uNAHgpWNu4IE4STa6tiq7FCweuxf5A/edit

Reply:
Har oppdatert nå, men det er uansett en midlertidig fiks. Kunne godt tenkt meg at vi begynte å se på samkjøring av lisenser(listen fra Johannes) og støtte for utvidet lisensvisning på backend snart.

Reply:
Jepp, enig. Hadde vært å få det på plass